tests/FSharp.Literate.Tests/DocContentTests.fs

```diff
@@ -225,3 +254,38 @@ let ``Parses frontmatter correctly `` () =
     twoTowersHtml |> shouldContainText "<a href=\"fellowship.html\">Previous</a>"
     twoTowersHtml |> shouldContainText "<a href=\"return.html\">Next</a>"
     returnHtml |> shouldContainText "<a href=\"two-tower.html\">Previous</a>"
+
+
+(* Cannot get this test to evaluate the notebook
```

Comment:
Why not? What goes wrong?

Reply:
I have `evaluate = true` set for DocContent, which should trigger the external call to `dotnet repl` that runs the notebook and produces outputs for the cell . But for some reason  the code calling out to `dotnet repl` isn't getting triggered inside this test. `dotnet fsdocs --eval` triggers notebook evaluation in a docs folder, but I have not been able to get it to work inside this test.

Reply:
Are you sure? This test is working for me in `WSL`.